### ISSUE TITLE
FlxInputText crashes on font change

### ISSUE THREAD

**Issue by ashes999** [HaxeFlixel/flixel-ui#168]
**TLDR:** repo is [here](https://github.com/ashes999/virconsole). Typing `trace` or `trace2` tries to set the `font` (or `setFormat`) respectively, and crashes.

Longer version: I'm trying to create something like a virtual console. I have a specific monospace font that I use in several rows of `FlxText` inputs, which I style like so:

```
this.rows[n] = new FlxText(0, n * this.fontSize);
this.rows[n].setFormat("assets/VeraMono.ttf", fontSize);`
```

I just discovered `FlxInputText`, which is superb; the best part is, it inherits from `FlxText`. So I tried this snippet in my FlxState `create` method:

```
this.input = new FlxInputText(0, (height - 2) * this.fontSize + PADDING_ADJUSTMENT, 800);
this.input.setFormat("assets/VeraMono.ttf", fontSize);
```

Nope, it crashes. Okay, how about poaching the font from one of my rows:

```
this.input.font = this.rows[0].font
```

Nope, still crashes. How do I resolve this?

Crash stacks are identical in both cases:

```
Invalid field access : __handle
Called from openfl/_legacy/display/Tilesheet.hx line 42
Called from a C function
Called from flixel/system/layer/TileSheetExt.hx line 22
Called from a C function
Called from flixel/system/layer/TileSheetExt.hx line 11
Called from flixel/system/layer/TileSheetData.hx line 47
Called from a C function
Called from flixel/system/layer/TileSheetData.hx line 19
Called from flixel/util/loaders/CachedGraphics.hx line 159
Called from flixel/FlxCamera.hx line 401
Called from flixel/FlxGame.hx line 808
Called from flixel/FlxGame.hx line 506
Called from openfl/_legacy/events/EventDispatcher.hx line 98
Called from a C function
Called from openfl/_legacy/display/DisplayObject.hx line 161
Called from a C function
Called from openfl/_legacy/display/DisplayObjectContainer.hx line 286
Called from openfl/_legacy/display/Stage.hx line 1103
Called from openfl/_legacy/display/Stage.hx line 351
Called from openfl/_legacy/display/Stage.hx line 1084
Called from openfl/_legacy/display/Stage.hx line 430
```

`haxe` reports itself as version 3.2.1. Flixel haxelibs below.

```
actuate: 1.7.5 [1.8.6]
flixel-addons: 1.1.0 [1.1.1]
flixel-demos: 1.1.1 [1.1.2]
flixel-templates: [1.0.2]
flixel-tools: 1.0.2 [1.1.2]
flixel-ui: [1.0.2]
flixel: [3.3.12] 3.3.6
hxcpp: [3.2.205] 3.1.39 3.2.193
lime-samples: [2.6.0]
lime: 2.0.0 [2.8.2]
openfl-bitfive: 2.1.6 [3.3.3]
openfl-samples: 2.1.0 [3.3.1]
openfl: [3.5.3]
svg: [1.0.9]
```

**Comment by ashes999** [comment]
I upgraded all my libraries via `haxelib upgrade` and it works now. Sorry about that!
